lib/EnsEMBL/REST/Model/Variation.pm

```diff
@@ -192,7 +192,7 @@ sub get_gene_phenotype_info {
 
   foreach my $phen (@pfs) {
     my $hash = $self->phen_as_hash($phen);
-    my $key = join("", sort values %$hash);
+    my $key = join("", sort grep {$_} values %$hash);
```

Comment:
This will need a comment, or someone will take it out in future since it appears to do nothing.